transport/awslambda/handler_test.go

```diff
@@ -21,6 +20,17 @@ const (
 	KeyEncMode   key = iota
 )
 
+// Created based on github.com/aws/aws-lambda-go/events.APIGatewayProxyRequest for the purposes of the tests below.
```

Comment:
Consider including the version of the github.com/aws/aws-lambda-go/events package referred to when creating these structs.

Reply:
Good point, done.